src/BBCodeParser.php

```diff
@@ -126,7 +126,7 @@ public function parse() {
 		$config = parse_ini_file('BBCodeParser.ini', true);
 		$options = $config['SSHTMLBBCodeParser'];
 
-		require_once(__DIR__ . '/../thirdparty/HTML_BBCodeParser2/BBCodeParser2.php');
```

Comment:
looks like it is: https://github.com/pear/HTML_BBCodeParser2/blob/master/composer.json#L24